.env

```diff
@@ -7,6 +7,9 @@ PUMA_THREADS=5
 PUMA_TIMEOUT=25
 PUMA_BOOT_TIMEOUT=25
 
+# credentials for "root" client that can modify allowed subscribers
+ROUTEMASTER_ROOT_KEY=CHANGE-ME
+
 # credentials for allowed clients (comma-separated)
 ROUTEMASTER_CLIENTS=demo
```

Comment:
It wasn't used at all, I was exporting it locally. Thought it was a good way to point out that it needs to be set up for anyone doing a new install - the same way many projects do in OOTB configs